apps/src/code-studio/pd/workshop_enrollment/enroll_form.jsx

```diff
@@ -309,6 +379,51 @@ export default class EnrollForm extends React.Component {
             />
           </FormGroup>
         }
+        {this.props.workshop_course === CSF && this.props.workshop_subject === DEEP_DIVE && <FormGroup>
+          <QuestionsTable
+            id="csf_course_experience"
+            key="csf_course_experience"
+            label="This workshop is designed for educators that have experience teaching CS Fundamentals. During the past year, how have you used CS Fundamentals course(s) with students?"
+            onChange={this.handleCsfCourseExperienceChange}
+            options={[
+              "a few lessons",
+              "most lessons",
+              "all lessons"
+            ]}
+            questions={Object.keys(CSF_COURSES).map(key =>
+              (
+                {label: CSF_COURSES[key],
+                name: key,
+            }))}
+            selectedItems={this.state.csf_course_experience}
+          />
+          <ButtonList
+            id="csf_courses_planned"
+            key="csf_courses_planned"
+            answers={csfCourses}
+            groupName="csf_courses_planned"
+            label={coursesPlannedLabel}
+            onChange={this.handleChange}
+            selectedItems={this.state.csf_courses_planned}
+            validationState={this.state.errors.hasOwnProperty("csf_courses_planned") ? VALIDATION_STATE_ERROR : null}
+            errorText={this.state.errors.csf_courses_planned}
+            type="check"
+          />
+          <ButtonList
+            id="attended_csf_intro_workshop"
+            key="attended_csf_intro_workshop"
+            answers={Object.keys(ATTENDED_CSF_COURSES_OPTIONS).map( key => key)}
```

Comment:
You don't need to do the `map` here - `Object.keys(ATTENDED_CSF_COURSES_OPTIONS)` returns the same thing

Reply:
Thanks! Fixed.